python/lsst/daf/butler/pydantic_utils.py

```diff
@@ -163,7 +163,9 @@ def validated(self, **kwargs: Any) -> _T:
             call to `validated`, *which will ignore ``**kwargs``*.
         """
         if not self._is_validated:
-            self._data = self._get_wrapped_type_adapter().validate_python(self._data, context=kwargs)
+            self._data = self._get_wrapped_type_adapter().validate_python(
+                self._data, strict=False, context=kwargs
```

Comment:
Without strict=False here, Pydantic wasn't doing any of its usual conversions like list -> tuple etc.